5G Mobile Dev Docs (#936)
helium/docs: docs/5g-on-helium/mining-5g.mdx

registered in the SAS.

:::

Install Helium Wallet app to manage HNT/Mobile token.

- [App Store](https://apps.apple.com/us/app/helium-hnt-wallet/id1609525848)
- [Google Play](https://play.google.com/store/apps/details?id=com.helium.wallet.app&gl=US)

Once the 5G rewards are activated, and given correct installation, the 5G Hotspot will start earning
MOBILE automatically.

:::tip

Be sure to update the Helium Wallet app to the latest version. More information can be found on the
[#announcements](discord://discord.com/channels/404106811252408320/730418759298318346) channel in
[Helium Discord](https://discord.gg/helium)

:::

## Emissions Curve

There will be a max supply of 250B MOBILE with issuance halvenings of every 2 years aligned with the
HNT issuance halvenings. The first halving will occur on August 1, 2023 to align with HNT halving
and will continue on a 2 year cycle afterward.

This allocation is distributed throughout the shortened 1 year period to online 5G Hotspots on an
epochal basis, weighted by Small Cell model.

:::info

More details on weighting will be added to this document in the future.

:::

> For clarity, the emission schedule is as follows:
> 
> |  Phase  | Weekly Emmision |
> | :-----: | --------------: |
> | Genesis |     640,000,000 |
> | 5G PoC  |   1,280,000,000 |
> 

Reviewer: ```suggestion
For clarity, the emission schedule is as follows:

|  Phase  | Weekly Emission |
| :-----: | --------------: |
| Genesis |     640,000,000 |
| 5G PoC  |   1,280,000,000 |

The Helium Foundation will work with the core developers to begin the Genesis period of MOBILE rewards. This will be paid out of the Foundation's pre-mined MOBILE rewards pool. The rate of rewards, as defined above, is half that defined in HIP 53 and will be proportionally distributed to all active 5G radios as described below.
```

of its activities on its own and will be rewarded for maintaining a valid connection to the SAS.

### Small Cell Radio Model Reward Scaling

All properly installed and registered 5G Hotspots will earn rewards for participating on the
Network. A rewards unit multiple is applied to the various Small Cell Models based on whether they
are indoor or outdoor, as well as their power output, as follows:

| Model           | Reward Scaler |  Small Cell Type   |
| --------------- | :-----------: | :----------------: |
| Nova 436H       |      2x       | High Power Outdoor |
| Nova 430i       |     1.5x      |      Outdoor       |
| Sercomm Outdoor |     1.5x      |      Outdoor       |
| Sercomm Indoor  |      1x       |       Indoor       |
| Neutrino 430    |      1x       |       Indoor       |

Inidividual rewards per period are calculated as a proporation of the total weighted rewards units.

### Indoor Small Cells

These less powerful Small Cell Radios are ideal for indoor installations, such as coffee shops,
bars, grocery stores, etc. Indoor Small Cell Radios are generally easier to install and create a
shorter range, localized area of 5G coverage.

> Indoor Small Cell Radios are physically smaller radios and can be installed at any height. It is

Reviewer: ```suggestion
Indoor Small Cell Radios are physically smaller radios and can be installed at any height.
```

---
id: mining-5g
hide_title: true
sidebar_label: 5G Mining & Rewards
sidebar_class_name: new
slug: /5g-on-helium/mining-5g
---

import useBaseUrl from '@docusaurus/useBaseUrl'

<img className="docsheader" src={useBaseUrl('img/5g-on-helium/mobile-header.png')} />

# 5G Mining and MOBILE Rewards

Several factors contribute to how much `MOBILE` a 5G Hotspot will earn over a given period, there
are some higher level concepts, design elements, and rules to keep in mind that will help better
explain what you might earn and why. This section covers these, as well as some mining and token
reward basics.

## How To Mine MOBILE

Mining MOBILE and providing 5G coverage requires a Helium Hotspot and a compatible CBRS Small Cell
Radio. These components can be purchased as a bundle or separately. 5G Hotspots must be online and
registered in the SAS (Spectrum Access System) maintained by the FCC. Specific instructions can be
found during onboarding from individual 5G Hotspot Makers.

:::warning

Misrepresentation of a CBRS radio’s installation information, including GPS location, height,
antenna orientation or other characteristics, in a SAS is a violation of FCC regulations and may

> result in fines or criminal penalties.

Reviewer 2: Are we sure about "criminal penalties"?

Reviewer: change to and/or but yes, apparently: https://www.ecfr.gov/current/title-47/chapter-I/subchapter-D/part-95/subpart-A/section-95.313

:::

:::note

Once the Hotspot is onboarded and Small Cell is physically installed and connected to the Hotspot,
it should automatically appear in the Radio Dashboard on the local IP address.

[http://freedomfi-gateway.local/](http://freedomfi-gateway.local/)

5G Hotspots will appear on explorer.helium.com between 2-4 days, depending on how quickly it is
registered in the SAS.

:::

> Install Helium Wallet app to manage HNT/Mobile token.

Reviewer 2: Please remind to minitor for updates and make sure users have the latest version, support for mobile is not released yet.

- [App Store](https://apps.apple.com/us/app/helium-hnt-wallet/id1609525848)
- [Google Play](https://play.google.com/store/apps/details?id=com.helium.wallet.app&gl=US)

Once the 5G rewards are activated, and given correct installation, the 5G Hotspot will start earning
MOBILE automatically.

## Emissions Curve

There will be a max supply of 250B MOBILE with issuance halvenings of every 2 years aligned with the
HNT issuance halvenings. The first halving will occur on August 1, 2023 to align with HNT halving
and will continue on a 2 year cycle afterward.

This allocation is distributed throughout the shortened 1 year period to online 5G Hotspots on an

> epochal basis, weighted by Small Cell model.

Reviewer 2: Please add "more details on weighting will be added to this document in the future"

|  48  | 06/05/2023 |      1.28       | 5G POC Start |     1.00     |   59.67    |
|  49  | 06/12/2023 |      1.28       | 5G POC Start |     1.00     |   60.96    |
|  50  | 06/19/2023 |      1.28       | 5G POC Start |     1.00     |   62.24    |
|  51  | 06/26/2023 |      1.28       | 5G POC Start |     1.00     |   63.52    |
|  52  | 07/03/2023 |      1.28       | 5G POC Start |     1.00     |   64.81    |
|  53  | 07/10/2023 |      1.28       | 5G POC Start |     1.00     |   66.09    |

## How Do Hotspots Earn Helium MOBILE Tokens?

The Helium blockchain rewards 5G Hotspots for providing 5G wireless coverage and verifying the
Helium Network. 5G Hotspots are rewarded in the Helium MOBILE Token.

During the Genesis Period, 5G Hotspots will earn MOBILE by reporting their status via a “heartbeat”,
an automatic process of reporting to the Blockchain that the Small Cell Radio is online and
registered with the SAS.

| Reward Type | Chain Variable | Description                                                             |
| :---------- | :------------- | :---------------------------------------------------------------------- |

> | Heartbeat   | `UNKNOWN`      | Rewarded to any 5G Hotspot that maintains a valid connection to the SAS |

Reviewer 2: There are three chainvars, let's drop this table for now.

Reviewer: agree on dropping. but maybe say below that we will start with "heartbeats" as a metric.

Author: dropping the larger emissions table or just the chain var table?

**_Do I Have To Actively Participate to Earn Rewards Once My 5G Hotspot is Deployed?_**

No. Once your 5G Hotspot is completely deployed and fully synced with the Helium blockchain, you as
the owner are not required to do anything else in order to earn MOBILE. Your 5G Hotspot will perform

> all of the above activities on its own while it runs.

Reviewer: update here based on comment above

### Indoor Small Cells

These less powerful Small Cell Radios are ideal for indoor installations, such as coffee shops,
bars, grocery stores, etc. Indoor Small Cell Radios are generally easier to install and create a
shorter range, localized area of 5G coverage.

Indoor Small Cell Radios are physically smaller radios and can be installed at any height. It is
Installing an Indoor Small Cell outdoors is not recommended as they are not weather resistant.

### Outdoor Small Cells

These Small Cell Radios are more Powerful than Indoor Small Cell Radios and are typically
weatherized to endure the elements. Outdoor Small Cell Radios create a larger area of 5G coverage
while requiring a bit more planning when installing and maintaining.

Outdoor Small Cells require installation above ground level and require registration with the
Spectrum Access System (SAS) to operate. Please be sure to evaluate your locations and ability to
install these Small Cell Radios, and consider hiring a professional to ensure your installation is
secure and meets all regulatory requirements.

> SAS registration is expected to be completed within 24 hours of submission, Hosts will be contacted

Reviewer 2: "Hosts" - might not be clear who is Host. Maybe stick with "you"

Reviewer: "Hotspot Owner"

---
id: mining-5g
hide_title: true
sidebar_label: 5G Mining & Rewards
sidebar_class_name: new
slug: /5g-on-helium/mining-5g
---

import useBaseUrl from '@docusaurus/useBaseUrl'

<img className="docsheader" src={useBaseUrl('img/5g-on-helium/mobile-header.png')} />

# 5G Mining and MOBILE Rewards

Several factors contribute to how much `MOBILE` a 5G Hotspot will earn over a given period, there
are some higher level concepts, design elements, and rules to keep in mind that will help better
explain what you might earn and why. This section covers these, as well as some mining and token
reward basics.

## How To Mine MOBILE

Mining MOBILE and providing 5G coverage requires a 5G Helium Hotspot and a compatible CBRS Small
Cell Radio. These components can be purchased as a bundle or separately. All 5G Hotspots must be
online and registered in the SAS (Spectrum Access System) maintained by the FCC to earn MOBILE
rewards.

Instructions on installing a CBRS Small Cell Radio and registration with the SAS is provided by the

> individual 5G Hotspot Makers. As an example, see the Freedomfi installation guides as a reference

Reviewer 2: Make sure FreedomFi is in camel-case everywhere.

here:

- [Freedomfi Indoor Install Guide](https://freedomfi.com/freedomfi-indoor-cbrs-small-cell-setup-guide/)
- [Freedomfi Outdoor Install Guide](https://freedomfi.com/freedomfi-outdoor-cbrs-small-cell-setup-guide/)

:::warning

Misrepresentation of a CBRS Radio installation information, including GPS location, height, antenna
orientation or other characteristics, in a SAS is a violation of FCC regulations and may result in
fines and/or criminal penalties.

:::

Once the 5G Hotspot is onboarded and Small Cell Radio is physically installed and connected to the
5G Hotspot, it should automatically appear in the Radio Dashboard on the local IP address.

> > [http://freedomfi-gateway.local/](http://freedomfi-gateway.local/)

Reviewer 2: After the Dashboard link, I would add something like "Once the Small Cell appears on the local Dashboard, you will be able to submit its installation details to register with SAS", for extra clarity.

During the Genesis Period, 5G Hotspots will earn MOBILE by reporting their status via a “heartbeat”,
an automatic process of reporting to the Blockchain that the Small Cell Radio is online and
registered with the SAS.

Once your 5G Hotspot is completely deployed and fully synced with the Helium blockchain, you as the
owner are not required to do anything else in order to earn MOBILE. Your 5G Hotspot will perform all
of its activities on its own and will be rewarded for maintaining a valid connection to the SAS.

## Emissions Curve

There will be a max supply of 250B MOBILE with issuance halvenings of every 2 years aligned with the
HNT issuance halvenings. The first halving will occur on August 1, 2023 to align with HNT halving
and will continue on a 2 year cycle afterward.

This allocation is distributed throughout the shortened 1 year period to online 5G Hotspots on an
epochal basis, weighted by Small Cell model.

For clarity, the emission schedule is as follows:

|  Phase  | Weekly Emission |
| :-----: | --------------: |
| Genesis |     640,000,000 |
| 5G PoC  |   1,280,000,000 |

The Helium Foundation will work with the core developers to begin the Genesis period of MOBILE

> rewards. This will be paid out of the Foundation's pre-mined MOBILE rewards pool. The rate of

Reviewer 2: "This will be paid out" might be read is if it references the work of core developers, but I might be just lacking some basic coffee.

> rewards, as defined above, is half that defined in HIP 53 and will be proportionally distributed to

Reviewer 2: This phrase doesn't seen to add clarity "The rate of rewards, as defined above, is half that defined in HIP 53".

Author: agreed, removing that sentence

all active 5G radios as described below.

## Small Cell Type Reward Scaling

All properly installed and registered 5G Hotspots will earn rewards for participating on the
Network. A rewards unit multiple is applied to the various Small Cell Models based on whether they
are indoor or outdoor, as well as their power output, as follows:

| Model           | Reward Weight | Small Cell Type    |
| :-------------- | :-----------: | :----------------- |
| Nova 436H       |     2.0x      | High Power Outdoor |
| Nova 430i       |     1.5x      | Outdoor            |
| Sercomm Outdoor |     1.5x      | Outdoor            |
| Sercomm Indoor  |     1.0x      | Indoor             |
| Neutrino 430    |     1.0x      | Indoor             |

> Individual rewards per period are calculated as a proporation of the total weighted reward units.

Reviewer 2: proporation > proportion ?

## Earnings Example

Earnings are calculated in 3 steps

1. Calculate the Total Emissions Weight and Emissions Weight Ratio for each Small Cell Type
2. Each Small Cell Type is allocated its ratio of the total rewards in the given epoch

> 3. Each 5G Hotspot is rewarded from its respective rewards pool by Small Cell Type

Reviewer 2: For extra clarity: "is rewarded" > "is proportionally rewarded"

As an example, we can calculate the rewards for a 5G Hotspot in a single epoch assuming the

> following:

Reviewer: ```suggestion
following (the below numbers are examples and may not reflect actual deployed Hotspots today):
```

> - 100,000 MOBILE Rewarded per epoch

Reviewer 2: For number of Mobile rewards, I would have a number that is close to what actual emission will be per epoch.

- 100 High Power Outdoor 5H Hotspots earn rewards
- 400 Outdoor 5G Hotspots earn rewards
- 500 Indoor 5G Hotspots earn rewards

### Reward Ratios

Calculate the Total Emissions Weight and Emissions Weight Ratio for each Small Cell Type:

> In this example, the Total Emissions Weight it 1,300 and the Ratios are as follows (rounded for ease

Reviewer 2: "Ratios" > "ratios"

of understanding)

```bash
  (High Power Outdoor Count * 2.0)    =>    200  =>  200 / 1,300  => 16%
+            (Outdoor Count * 1.5)    =>  + 600  =>  600 / 1,300  => 46%
+             (Indoor Count * 1.0)    =>  + 500  =>  500 / 1,300  => 38%
------------------------------------      ------
                                          1,300
```

### Small Cell Type Pool

> Each Small Cell Type is allocated its ratio of the total rewards in the given epoch

Reviewer 2: missing dot at the end

> Small Cell Type Reward Pool = Epoch Rewards \* Weighted Small Cell Type Ratio

```bash
High Power Outdoor  => 100,000 * 16% = 16,000 MOBILE
Outdoor             => 100,000 * 46% = 46,000 MOBILE
Indoor              => 100,000 * 38% = 38,000 MOBILE
```

### Rewards Per 5G Hotspot

> Each 5G Hotspot is rewarded from its respective rewards pool by Small Cell Type

Reviewer 2: same as above
